src/styles/blocks/section.scss

```
@@ -0,0 +1,27 @@
+.main {
+  --column-count: 1;
+  display: grid;
+  grid-template-columns: repeat(var(--column-count),200px);
+  column-gap: 48px;
+  row-gap: 46px;
+  justify-content: center;
+  padding: 50px 40px 72px;
+
+  @media (min-width: 488px) {
+    & {
+      --column-count: 2;
+    }
+  }
```

```suggestion
  @media (min-width: 488px) {
    --column-count: 2;
  }
```
You don't need to repeat selector

---

Not fixed